taxcalc/calculate.py

```diff
@@ -160,6 +160,36 @@ def increment_year(self):
         self.policy.set_year(self.policy.current_year + 1)
         self.behavior.set_year(self.policy.current_year)
 
+    def year_increment(self, advance_to_year):
+        '''
+        The year_increment function gives an optional way of implementing
+        increment year functionality by immediately specifying the year
+        as input. Robustness has been added to ensure input is of good shape.
+        '''
+        if isinstance(advance_to_year, int):
```

Comment:
In Python, we typically try to avoid type checking unless we really need to. Here I think it is not necessary.